solidity/contracts/system/TBTCSystem.sol

```diff
@@ -173,6 +173,8 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     /// @notice One-time-use emergency function to disallow future deposit creation for 10 days.
     function emergencyPauseNewDeposits() external onlyOwner {
         require(pausedTimestamp == 0, "emergencyPauseNewDeposits can only be called once");
+        uint256 diff = block.timestamp - initializedTimestamp;
```

Comment:
I mean I’ll say if you’re going to pull this out of the context of the require, giving it a super generic name like `diff` is kinda meh, but we can roll with it 😉